src/templates/Toolbar.html

```diff
@@ -1,38 +1,37 @@
 <div class="toolbar" id="toolbar">
-  <div>
-    <img class="logo" src="~images/logo.png" />
+  <div class="top-nav">
+    <img class="logo" src="~images/logo.png" alt="logo"/>
     <h2 class="cta">Find events near you</h2>
   </div>
-  <div>
+  <div class="event-filters">
+
     <input
       type="text"
       @input="updateZipcode"
       placeholder="Search by zip code"
       :value="zipcode"
       autofocus
     />
-    <button type="button" class="btn -primary">
-      <i class="icon-search"></i>
-    </button>
-  </div>
-  <div>
-    <button
-      type="button"
-      class="btn -fixed-width"
-      @click="toggleFilterEvents"
-    >
-      Filter events
-    </button>
-    <button
-      type="button"
-      class="btn -fixed-width"
-      @click="toggleView"
-    >
+
+    <button class="btn date-picker" @click="toggleFilterEventsByDate"><span>DATE OF EVENT</span> &nbsp;&nbsp;<img id="date-arrow" class="arrow" src="~images/down-arrow.svg"/></button>
```

Comment:
Yeah that would definitely be better!  I added the ids so that we could toggle the img src inside them to flip the caret when you click, but I couldn't quite get it working so I didn't commit it.